lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -113,6 +135,15 @@ def custom_transform
         def ignored_words
           cop_config.fetch('IgnoredWords', [])
         end
+
+        def insufficient_examples
+          examples = cop_config.fetch('DisallowedExamples', [])
+          examples.map! { |example| preprocess(example) }
```

Comment:
Won't this line allocate (at least) 3 String objects on each `on_block` call? I don’t know if it would be possible to memoize this at class level.

Reply:
Since, in theory, different `DisallowedExamples` can be configured in different directories, it would be incorrect to cache this at the class level.
I've made the change to cache it in an instance variable, as I guess the same instance will be used for a single investigation.
We also need to [nullify the instance variable in `on_new_investigation`](https://docs.rubocop.org/rubocop-rspec/upgrade_to_version_2.html#always-call-super-from-on_new_investigation-in-your-cops). Is it worth it?

Reply:
No, probably not worth it. Thanks for investigating.